tick/array_test/tests/array_memory_test.py

```diff
@@ -259,15 +259,15 @@ def get_memory_used():
                 gc.collect()
                 filled_memory = get_memory_used()
                 self.assertAlmostEqual(filled_memory, initial_memory,
-                                       delta=1.1 * bytes_size)
+                                       delta=2.1 * bytes_size)
 
                 X = tick_double_sparse2d_from_file(cereal_file)
                 del X
 
             gc.collect()
 
             end = get_memory_used()
-            self.assertAlmostEqual(end, initial_memory, delta=1.1 * bytes_size)
```

Comment:
Why this high :( ?

Reply:
tests fail

Reply:
we can make a ticket to investigate it

Reply:
https://github.com/X-DataInitiative/tick/issues/332

Reply:
That sounds frightening... 
It might mean that each time we create an array we have a memory leak...

Reply:
we haven't changed anything, and it doesn't always happen so I dunno what to tell you

Reply:
I was running python 3.5 so let me test some more

Reply:
works on CI now resetting back to 1.1 delta